tests/unit/scripts/test_ask_qdrant.py

```
@@ -95,26 +135,82 @@ def embedding(self, *, input_text: str):  # noqa: ARG002 - signature parity
+            
+            Stores the passed positional and keyword arguments into the shared `capture` mapping under keys `"args"` and `"kwargs"`, and creates `FakeRecord` instances from the surrounding `records` sequence, assigning them to `self._records`.
+            
+            Parameters:
+                *args: Positional arguments supplied to the retriever constructor; captured in `capture["args"]`.
+                **kwargs: Keyword arguments supplied to the retriever constructor; captured in `capture["kwargs"]`.
+            """
+            capture["args"] = args
+            capture["kwargs"] = kwargs
+            self._records = [FakeRecord(item) for item in records]
+
+        def get_search_results(self, *, query_vector, top_k):  # noqa: ARG002
```

Ruff (0.13.2)

Unused `noqa` directive (unused: `ARG002`)

Remove unused `noqa` directive

(RUF100)